# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/epplets-extra16/e-cdplay.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ecdplay/COMPILE.orig-rock-maint	2003-08-29 10:19:30.000000000 +0200
+++ ecdplay/COMPILE	2003-08-29 10:20:03.000000000 +0200
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-cc ecdplay.c epplet.c -o ecdplay.epplet `imlib-config --libs` `imlib-config --cflags` -lcdaudio
+$CC $CFLAGS ecdplay.c epplet.c -o ecdplay.epplet `imlib-config --libs` `imlib-config --cflags` -L$E16/lib -lepplet -lcdaudio -lGL
 
 cp -f ecdplay.epplet $EBIN/ecdplay.epplet
 chmod a+x $EBIN/ecdplay.epplet
